app.js

```diff
@@ -4,16 +4,32 @@
  */
 import express from 'express';
 import logger from 'morgan';
+import session from 'express-session';
 
-import userInfosHelper from './helpers/userInfo';
-import idHintHelper from './helpers/idHintToken';
-import logoutHelper from './helpers/logout';
-import authorizationHelper from './helpers/authorization';
+import utilsHelper from './helpers/utils';
 import getAccessTokenHelper from './helpers/accessToken';
```

Comment:
please use destructured import to avoid useless variables like "getAccessTokenHelper":

```js
import { getAccessToken } from './helpers/utils';
```